src/modals/reveal-seed-phrase-modal/reveal-seed-phrase-form-content/reveal-seed-phrase-form-content.tsx

```diff
@@ -28,7 +28,9 @@ export const RevealSeedPhraseFormContent: FC<FormikProps<RevealSeedPhraseModalFo
     return getDerivationPath(currentAccountIndex);
   }, [hdAccounts, values.account.publicKeyHash]);
 
-  useEffect(() => setFieldValue('derivationPath', derivationPath), [derivationPath]);
+  useEffect(() => {
+    setFieldValue('derivationPath', derivationPath);
+  }, [derivationPath]);
```

Comment:
try `useEffect(() => void setFieldValue('derivationPath', derivationPath), [derivationPath]);`  
would this work?

Reply:
it works